<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoffeShop</class>
 <widget class="QMainWindow" name="CoffeShop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CoffeShop</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#CB925D;
color:white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="Resource.qrc">:/icon/image/favicon.webp</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Impact</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CoffeShop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">
background-color:#46200B;
border:none;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="stackedWidgetPage1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item alignment="Qt::AlignTop">
         <widget class="QFrame" name="frame_4">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="lineEdit">
             <property name="font">
              <font>
               <family>Impact</family>
               <pointsize>-1</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">height:40px;
background-color:#CB925D;
color:white;
font-size:16px;
padding:0 5px;</string>
             </property>
             <property name="placeholderText">
              <string>Search ...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="font">
              <font>
               <family>Impact</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">height:40px;
background-color:#CB925D;
padding:0 5px;</string>
             </property>
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="stackedWidget_5">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QWidget" name="stackedWidget_5Page1">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QScrollArea" name="ShowCart">
              <property name="styleSheet">
               <string notr="true">QFrame{
	background-color:#D9D9D9;
	color:black;
}

QFrame&gt;QPushButton{
	padding:3px;
}</string>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>746</width>
                 <height>271</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4"/>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="completePurchase">
             <property name="font">
              <font>
               <family>Impact</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">width:100px;
height:40px;
padding:0 5px;
background-color:#CB925D;</string>
             </property>
             <property name="text">
              <string>Purchase</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="AllProductType">
         <property name="font">
          <font>
           <family>Impact</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:#DEDEDE;
padding:5px 0;
color:black;
font-size:16px;
border:none;</string>
         </property>
         <property name="text">
          <string>All Product</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="coldType">
         <property name="font">
          <font>
           <family>Impact</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:#DEDEDE;
padding:5px 0;
color:black;
font-size:16px;
border:none;</string>
         </property>
         <property name="text">
          <string>Cold Drink</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="hotType">
         <property name="font">
          <font>
           <family>Impact</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:#DEDEDE;
padding:5px 0;
color:black;
font-size:16px;
border:none;</string>
         </property>
         <property name="text">
          <string>Hot Drink</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="foodType">
         <property name="font">
          <font>
           <family>Impact</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:#DEDEDE;
padding:5px 0;
color:black;
font-size:16px;
border:none;</string>
         </property>
         <property name="text">
          <string>Food</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
